packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2380,7 +2399,17 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
   int? get currentValueLength => _currentValueLength;
   int? _currentValueLength;
 
-  bool _canPerformAction(SemanticsAction action) => _actions.containsKey(action);
+  /// The level of the widget (when it's defined as a header) within
```

Comment:
First paragraph should be a brief sentence also this can use more explanation. can this parameter make sense out side of dom or markdown? If not, we should add document in terminologies in dom or markdown

Reply:
Sure, I tried to add more specific explanation here, please let me know if I should add/change something else. Thank you!